packages/@aws-cdk/service-spec-importers/src/types/registry-schema/JsonSchema.ts

```diff
@@ -15,14 +17,18 @@ export namespace jsonschema {
 
   export type EmptyObject = Record<string, never>;
 
-  export function isAnyType(x: Schema): x is AnyType {
+  export function isAnyType(x: Schema | CommonTypeCombinatorFields): x is AnyType {
```

Comment:
Why is `CommonTypeCombinatorFields` needed here?

Reply:
Because the entries in a `oneOf` list can be a `CommonTypeCombinatorFields`, and we check those entries with `isAnyType` below.